SDWebImage/UIView+WebCacheOperation.m

```diff
@@ -30,7 +30,7 @@ - (SDOperationsDictionary *)sd_operationDictionary {
 }
 
 - (void)sd_setImageLoadOperation:(nullable id<SDWebImageOperation>)operation forKey:(nullable NSString *)key {
-    if (key) {
+    if (key && operation) {
```

Comment:
Hi bro, How about removing this judgment in 35 lines？
```Objc 
if (operation) {            

 }
```

Reply:
Does this cause issue ? I think this implementation is correct. When the operation is nil, trigger a cancel and remove instead.

Because currently the code we use does not use that `sd_cancelImageLoadOperationWithKey:`, we only call through the `sd_setImageLoadOperation:forKey:` with nil operation.

Reply:
@dreampiggy I overlooked one line, so the crash issue I think is not exist. Please see my [comment](https://github.com/SDWebImage/SDWebImage/pull/2536#discussion_r234550755),  if it's unnecessary, we can only fix the typo.

Reply:
I'll fix the comment here soon. Seems it was not sync to the code changes.